src/main/resources/assets/wurst/lang/zh_cn.json

```diff
@@ -1,149 +1,170 @@
-  "description.wurst.hack.snowshoe": "可以让你在细雪上行走。",
-  "description.wurst.hack.speedhack": "允许你以大约 ~2.5x 或更快的速度跳跃或疾跑。\n\n§6§l警告：§r只有低于反作弊版本 3.13.2 才会工作且绕过，如果你不知道服务器反作弊是哪个版本。\n输入 §l/ncp version§r 来查看反作弊版本。",
-  "description.wurst.hack.speednuker": "一个比 Nuker 更快的版本，但会无法绕过反作弊。",
-  "description.wurst.hack.spider": "像蜘蛛一样能够爬墙。",
-  "description.wurst.hack.step": "允许你走上更高高度（取决于你的设定）的格子方块。",
-  "description.wurst.hack.throw": "迅速丢你手中种类的物品，可以用于丢雪球、鸡蛋、刷怪蛋、矿车，诸如此类。\n\n会生成大量的数量，这会产生大量的 lag 且可能会导致服务器崩溃。",
-  "description.wurst.hack.tillaura": "自动将泥土、草块、诸如此类。转化为耕地。\n不要和 Killaura 搞混。",
+  "description.wurst.hack.sneak": "保持潜行。",
+  "description.wurst.hack.snowshoe": "允许你在细雪上行走。",
+  "description.wurst.hack.speedhack": "允许你以大约 2.5x 或更快的速度疾跑和跳跃。\n\n§6§l警告：§r只在反作弊版本低于 3.13.2 的服务器有效，如果你不知道服务器反作弊是哪个版本。\n输入 §l/ncp version§r 来查看反作弊版本。",
+  "description.wurst.hack.speednuker": "一个比 Nuker 更快的版本，但无法绕过反作弊。",
+  "description.wurst.hack.spider": "允许你像蜘蛛一样爬墙。",
+  "description.wurst.hack.step": "允许你走上更高（取决于你的设定）的方块。（类似你走上半砖的效果）",
+  "description.wurst.hack.throw": "多次使用你手中的物品，可以用于丢雪球、鸡蛋、刷怪蛋、矿车，诸如此类。\n\n会生成大量的数量，这会产生巨大的延迟且可能导致服务器崩溃。",
+  "description.wurst.hack.tillaura": "自动将泥土、草块等方块转化为耕地。\n不要和 Killaura 搞混。",
   "description.wurst.hack.timer": "改变大部分东西的速度。",
   "description.wurst.hack.tired": "让你看起来很像 2015 年 4 月的 Alexander。\n仅限其他玩家可视。",
-  "description.wurst.hack.toomanyhax": "屏蔽那些你并不想要有的功能。\n让你确保你不会不小心开启错误的作弊功能导致被服务器封禁。\n这个是给那些 “只想作弊一点点的用户”。\n\n使用指令 §6.toomanyhax§r 来选择哪些功能需要被屏蔽。\n输入 §6.help toomanyhax§r 寻求更多。",
-  "description.wurst.hack.tp-aura": "自动传送到实体附近并自动攻击他。",
+  "description.wurst.hack.toomanyhax": "屏蔽那些你并不想要有的功能。\n确保你不会不小心开启错误的作弊功能导致被服务器封禁。\n这个是给那些 “只想轻度作弊的用户”。\n\n使用指令 §6.toomanyhax§r 来选择哪些功能需要被屏蔽。\n输入 §6.help toomanyhax§r 查看详细信息。",
+  "description.wurst.hack.tp-aura": "自动传送到实体附近并自动攻击。",
   "description.wurst.hack.trajectories": "预测箭和投掷物品的飞行路径。",
-  "description.wurst.hack.treebot": "一个实验性的机器人将会在树木附近砍树。\n目前仅限于砍小树。",
-  "description.wurst.hack.triggerbot": "自动攻击你所看的实体。",
-  "description.wurst.hack.trollpotion": "生成一个惹人恼怒的药水效果。\n§o需要创造模式。§r",
+  "description.wurst.hack.treebot": "实验性的自动砍树机器人。\n目前仅限于砍小树。",
+  "description.wurst.hack.triggerbot": "自动攻击准星所指的实体。",
+  "description.wurst.hack.trollpotion": "生成一个恼人的药水效果。\n§o需要创造模式。§r",
   "description.wurst.hack.truesight": "允许你看到隐身的实体。",
-  "description.wurst.hack.tunneller": "自动挖一个隧道。\n\n§c§l警告：§r 尽管这个 AI 会自动避开岩浆和其他危险，无法确保你不会死亡，除非你不介意你有一定概率失去你的全部物品。",
-  "description.wurst.hack.x-ray": "允许你透视矿物所在的位置（请自行确保服务器内无假矿插件）。",
+  "description.wurst.hack.tunneller": "自动挖一个隧道。\n\n§c§l警告：§r 尽管这个 AI 会自动避开岩浆和其他危险，但无法确保你不会死亡，除非你不介意一定概率失去你的全部物品。",
+  "description.wurst.hack.x-ray": "透视矿物所在位置（请自行确保服务器内无假矿插件）。",
+  "description.wurst.setting.generic.attack_speed": "以每秒点击次数计算的攻击速度\n0 = 根据你的攻击冷却时间动态调整攻击速度。",
+  "description.wurst.setting.generic.pause_attack_on_containers": "Won't attack while a container screen (chest, hopper, etc.) is open.当一个容器界面（如箱子、漏洞等）打开时不会攻击。\n在一些显示容器界面作为菜单的小游戏服务器很有用。",

```

Comment:
Won't attack while a container screen (chest, hopper, etc.) is open.
这一段英文要移除，你后面已经汉化了。
This paragraph should be removed,because you already translate it.

Reply:
It's fixed.